README.md

```diff
@@ -1 +1,502 @@
+
+
+![alt text](https://github.com/pranavan15/service-composition/blob/master/images/serviceComposition.png)
+
+
+
+## <a name="pre-req"></a> Prerequisites
+ 
+- JDK 1.8 or later
+- [Ballerina Distribution](https://ballerinalang.org/docs/quick-tour/quick-tour/#install-ballerina)
+- A Text Editor or an IDE 
+
+Optional Requirements
+- Ballerina IDE plugins (IntelliJ IDEA, VSCode, Atom)
+
+## <a name="developing-service"></a> Developing the Service
+
+### <a name="before-begin"></a> Before You Begin
+##### Understand the Package Structure
+Ballerina is a complete programming language that can have any custom project structure as you wish. Although language allows you to have any package structure, we'll stick with the following package structure for this project.
+
+```
+service-composition
+├── TravelAgency
+│   ├── AirlineReservation
+│   │   ├── airline_reservation_service.bal
+│   │   └── airline_reservation_service_test.bal
+│   ├── CarRental
+│   │   ├── car_rental_service.bal
+│   │   └── car_rental_service_test.bal
+│   ├── HotelReservation
+│   │   ├── hotel_reservation_service.bal
+│   │   └── hotel_reservation_service_test.bal
+│   ├── travel_agency_service.bal
+│   └── travel_agency_service_test.bal
+└── README.md
+
+```
+
+Package `AirlineReservation` contains the service that provides online flight ticket reservations.
+
+Package `CarRental` contains the service that provides online car rentals.
+
+Package `HotelReservation` contains the service that provides online hotel room reservations.
+
+`travel_agency_service.bal` file provides travel agency service, which consumes the other three services and arranges a complete tour for the requested user.
+
+
+### <a name="Implementation"></a> Implementation
+
+Let's get started with the development of `travel_agency_service.bal` file, which includes the implementation of the Travel agency service. This is the service that acts as the composition initiator. It gets requests from users for tour arrangements and communicates with other necessary services to successfully arrange a complete journey for the user. Refer the code attached below. Inline comments are added for better understanding.
```

Comment:
Can we break the implementation so that we highlight the steps involved in creating a service composition with multiple services.

Reply:
This requested change has been done in the following PR.

PR: https://github.com/ballerina-guides/service-composition/pull/4